packages/block-library/src/social-link/block.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "core/social-link",
+	"category": "widgets",
+	"attributes": {
+		"url": {
+			"type": "string"
+		},
+		"site": {
```

Comment:
Is "site" the best attribute name?

Reply:
What about `service`?

Reply:
I've wondered about `service` too. Could try that.

Reply:
How's this, @mtias? ba9070f